### PR TITLE
Add info logging to ThreadRunner

### PR DESCRIPTION
**Pull request by pascalwhoop**
## Description
To add more node completion information to the threadrunner.

## Development notes
It's only logging lines. 

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: kedro/runner/thread_runner.py [hunk 1]
```diff
@@ -147,6 +147,10 @@ def _run(  # pylint: disable=too-many-locals,useless-suppression
                         self._suggest_resume_scenario(pipeline, done_nodes)
                         raise
                     done_nodes.add(node)
+                    self._logger.info("Completed node: %s", node)
```
Comment (lorenabalan) [inline kedro/runner/thread_runner.py:150]: Wouldn't this be more readable? 

```suggestion
                    self._logger.info("Completed node: %s", node.name)
```

**Review by lorenabalan — APPROVED** [review]
LGTM

**Review by MerelTheisenQB — APPROVED** [review]
👍

File: kedro/runner/thread_runner.py (hunk 1 above)
Comment (pascalwhoop) [reply kedro/runner/thread_runner.py]: Indeed, ha I never noticed that it's so verbose throughout the project 🙈

**Comment by pascalwhoop** [comment]
@lorenabalan any way I can retrigger the linting? it fails on 
```
/home/circleci/project/docs/build/kedro.extras.datasets.geopandas.GeoJSONDataSet.rst.rst:2:broken link: https://geopandas.org/reference.html#geopandas.GeoDataFrame.to_file (Anchor 'geopandas.GeoDataFrame.to_file' not found)
``` 
which seems either unrelated or like a random error

**Comment by deepyaman** [comment]
Can you add your name to release notes, and--the reason why I came looking for a ready PR ;)--fix the missing comma here while you're at it:

image

Pretty please? 🍒

**Comment by pascalwhoop** [comment]
@deepyaman done

**Pull request merged**